lib/index.js

```diff
@@ -334,6 +334,20 @@ Mixpanel.prototype._parseResponse = function(fn){
   return this.handle(function(err, res){
     if (err) return fn(err);
     if (!res.body.status) return fn(self.error(res.body.error));
+    
+    if (res.body.text) {
```

Comment:
Is text only available when there's an error? (It may, I'm just not familiar with their API)